pages/privacy.js

```diff
@@ -0,0 +1,254 @@
+Examples of Cookies we use: session cookies (we use these cookies to operate our Service) and preference cookies (we use these cookies to remember your preferences and various settings).
+          </ListItem>
+        </UnorderedList>
+        <Divider />
+        <FAQHeading>Use of Data</FAQHeading>
+        <PrivacyTextBlock>We use the collected data for various purposes:</PrivacyTextBlock>
+        <UnorderedList>
+          <ListItem>
+          To provide and maintain our Service
+          </ListItem>
+          <ListItem>To notify you about changes to our Service</ListItem>
+          <ListItem>To allow you to participate in interactive features of our Service when you choose to do so</ListItem>
+          <ListItem>To provide customer support</ListItem>
+          <ListItem>To gather analysis or valuable information so that we can improve our Service</ListItem>
+          <ListItem>To monitor the usage of our Service</ListItem>
+          <ListItem>To detect, prevent and address technical issues</ListItem>
+        </UnorderedList>
+        <Divider />
+        <FAQHeading>Retention of Data</FAQHeading>
+        <PrivacyTextBlock>
+        We will retain your Personal Data only for as long as is necessary for the purposes set out in this Privacy Policy. We will retain and use your Personal Data to the extent necessary to comply with our legal obligations (for example, if we are required to retain your data to comply with applicable laws), resolve disputes, and enforce our legal agreements and policies.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+        We will also retain Usage Data for internal analysis purposes. Usage Data is generally retained for a shorter period of time, except when this data is used to strengthen the security or to improve the functionality of our Service, or we are legally obligated to retain this data for longer time periods.
+        </PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>Transfer Of Data</FAQHeading>
+        <PrivacyTextBlock>
+        Your information, including Personal Data, may be transferred to — and maintained on — computers located outside of your state, province, country or other governmental jurisdiction where the data protection laws may differ than those from your jurisdiction.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+        We will take all steps reasonably necessary to ensure that your data is treated securely and in accordance with this Privacy Policy and no transfer of your Personal Data will take place to an organization or a country unless there are adequate controls in place including the security of your data and other personal information.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+        Your consent to this Privacy Policy followed by your submission of such information represents your agreement to that transfer.
+        </PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>Disclosure Of Data</FAQHeading>
```

Comment:
don't these 2 exceptions fly in the face of the TLDR?

we will never share your data... unless someone buys us... or the government asks us.

Reply:
i don't think they do. we won't share your data. if we're bought, your data will be transferred and if we're required by law to turn over our db's, we will not break the law. 

i think we can never avoid having to share data if required by law or if we're bought. that being said, i think that's all the more reason to keep the data we store to the bare minimum, which is what we're doing.